Source/WebCore/html/HTMLFormControlElement.cpp

```diff
@@ -395,14 +395,19 @@ void HTMLFormControlElement::setPopoverTargetAction(const AtomString& value)
 }
 
 // https://html.spec.whatwg.org/#popover-target-attribute-activation-behavior
-void HTMLFormControlElement::handlePopoverTargetAction() const
+void HTMLFormControlElement::handlePopoverTargetAction(const EventTarget* eventTarget) const
 {
     RefPtr target = popoverTargetElement();
```

Comment:
Can you rename this variable to `popover` to make the logic less confusing to read?